docs/content/integrations/dbt/using-dbt-with-dagster-plus.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: "Using dbt with Dagster+ | Dagster Docs"
+description: Deploy your dbt & Dagster project in Dagster+.
+---
+
+# Using dbt with Dagster+
+
+Importing an existing dbt project in Dagster+ allows you to automatically load your dbt models as Dagster assets. This can be be done with:
+
+- An existing dbt project that is not already using Dagster, or
+- A Dagster project in which your dbt project is included
+
+In this guide, we'll demonstrate by using an existing dbt project that doesn't use Dagster.
+
+---
+
+## Prerequisites
+
+To follow the steps in this guide, you'll need **Dagster+ Organization Admin**, **Admin**, or **Editor** permissions. This is required to create a code location.
+
+You'll also need **an existing dbt project** that contains the following files in the repository root:
+
+- [`dbt_project.yml`](https://docs.getdbt.com/reference/dbt_project.yml)
+- [`profiles.yml`](https://docs.getdbt.com/docs/core/connect-data-platform/profiles.yml)
+
+---
+
+## Step 1: Import your project in Dagster+
+
+<Note>
+  <strong>Heads up!</strong> This guide focuses on Serverless deployments in
+  Dagster+, but dbt projects may also be deployed if using a Hybrid deployment.
+  For more information about Hybrid deployments, refer to the{" "}
+  <a href="/dagster-plus/deployment/hybrid#hybrid-deployments-in-dagster">
+    Hybrid deployment documentation
+  </a>
+  .
+</Note>
+
+In this section, we'll demonstrate how to import an existing project Dagster+. Our example imports the project from a GitHub repository, but Dagster+ also supports Gitlab.
```

Comment:
@erinkcochran87 Could we mention here that we can import either a dbt project or a dbt & Dagster project? I think consolidating both sections in one makes sense, but maybe we want to be explicit about the option of importing an existing dbt & Dagster project?

Reply:
Right, in that case I think that's ok.

A small typo though

```suggestion
In this section, we'll demonstrate how to import an existing project to Dagster+. Our example imports the project from a GitHub repository, but Dagster+ also supports Gitlab.
```

Reply:
Good catch!